src/controllers/hid/hidcontroller.cpp

```diff
@@ -295,6 +242,23 @@ int HidController::close() {
     return 0;
 }
 
+bool HidController::poll() {
+    Trace hidRead("HidReader read packet");
+    int result = hid_read(m_pHidDevice, m_pPollData, 255);
```

Comment:
`int result = hid_read(m_pHidDevice, m_pollData, sizeof(m_pollData) / sizeof(m_pollData[0]));`

Using static allocation the capacity is implicitly defined.

Reply:
Good point.